docs/Metrics/DeployTime.md

```diff
@@ -16,11 +16,26 @@ The time it takes from when a PR is merged to when it is deployed.
 
 
 ## How is it calculated?
-You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+1. You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+2. Connect pr/mr to the right `deployment`
+![](/img/Metrics/pr-commit-deployment-relation.jpeg)
+   - Find the [commits diff](the link to refdiff plugin) between two consecutive deployments by deployments' commit_sha 
+   - under the same scope and environment (in terms of TESTING/STAGING/PRODUCTION)
+for example, here, we will get commit-2 and commit-3 by calculating commits_diff before deployment-1 and deployment-2,
+we can say commit-2, commit-3 and commit-4 are deployed by deployment-2
+   - Connect pr/mr and commits_diff through merge_commit or pr_commit, for example, 
```

Comment:
Please capitalize PR/MR to make them consistent.

Reply:
fixed